app/policies/document_policy.rb

```diff
@@ -8,6 +8,8 @@ def index?
   alias_method :edit?, :index?
   alias_method :update?, :index?
   alias_method :show?, :index?
+  #TODO fix this, attachments are using the wrong policy
```

Comment:
OK I don't think this should block this work but it would be good to understand why the wrong policy is in use and add the reason to a FIXME comment so we can revisit this.